packages/@react-aria/select/src/HiddenSelect.tsx

```diff
@@ -88,7 +88,7 @@ export function useHiddenSelect<T>(props: AriaHiddenSelectOptions, state: Select
     containerProps: {
       ...visuallyHiddenProps,
       'aria-hidden': true,
-      ['data-hidden-select-ignore']: true,
+      ['data-rsp-prevent-focus']: true,
```

Comment:
should it be `data-react-aria-prevent-focus`? its not rsp specific

Reply:
ah, yup lemme update that